### DISCUSSION TITLE
plot_response_curves

### DISCUSSION THREAD

**Discussion by SandraLG3** [google/lightweight_mmm#182] [General]
Hi!

I'm using lightweight_mmm v0.1.7. When I try to execute this cell on colab
![image](https://user-images.githubusercontent.com/126599044/221916421-f428ff0a-edd9-40fc-8cc4-4b896840d1ce.png)
I obtain this error
![image](https://user-images.githubusercontent.com/126599044/221916699-3124aaf0-1ed2-4d26-a06d-70c663892504.png)

Nevertheless, lighweight_mmm v0.1.3 runs perfectly!

I don't get solve it! Can someone help me please? Thank you very much!

**Comment by hansbaumberger18** [comment]
Hey Sandra, I am also using lightweight_mmm v0.17. Haven't you had issues with numba's and numpy's versions? I installed numba v 0.56.4 which is not compatible with numpy v 1.24.2 (but this numpy v is required to work with lightweight_mmm v 0.17). What versions did you use so the imports wouldn't result in ERRRORs?

**Comment by biggielutz** [comment]
I am getting the same error- even when i use the older version. Has anybody found a solution to this?

**Comment by biggielutz** [comment]
i found the solution in the the discussions group- need to add this code

!pip uninstall -y jax
!pip install jax==0.4.2

!pip uninstall -y jaxlib
!pip install jaxlib==0.4.2

**Reply by hansbaumberger18** [reply]
I am still encountering errors after adding this code. Did it actually work for you, and if so, could you upload the imports and pip install code snippets in their order? thankyou
